playground/src/DevPlayground/LiteInsight/AutoInsight.tsx

```diff
@@ -1,19 +1,19 @@
-import React, { useState, useEffect, createRef } from 'react';
+import React, { useState, useEffect } from 'react';
```

Comment:
Is the site demo in the next PR?

Reply:
yes, site demos and docs are not included in this PR